tests/e2e/setup.sh

```diff
@@ -71,6 +71,58 @@ beforeAll
 ######################################################################
 #
 
+name="interactive setup"
```

Comment:
Can you prepend the `expect` commit with `chore:` so that it doesn't show up in the changelog?